diesel_cli/src/config.rs

```diff
@@ -64,3 +66,15 @@ impl PrintSchema {
         self.import_types.as_ref().map(|v| &**v)
     }
 }
+
+#[derive(Default, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct MigrationsDirectory {
+    pub file: PathBuf,
+}
+
+impl MigrationsDirectory {
+    pub fn file(&self) -> &PathBuf {
+        &self.file
+    }
+}
```

Comment:
Thanks for the review. I wonder what's better between a direct access to the field or through a function?

Reply:
Depends on what you want to achieve, namely:

A field access allows a super set of the accessor function (i.e. you're not really trying to maintain some state). For example, if you pass an `&mut` to the struct, you can replace the file field with an alternative. If you pass an `&` to the struct, it will be almost the same as an accessor method.

A function allows you to either hide some of the complexity or return a conglomeration/derived field as well. This doesn't seem necessary here. An analogous case is present in the struct abobe. If we look there, I believe file is treated as a field and there is no accessor function for the field. This is because it is solely meant to hold data and it's not pertinent to hide the details behind the field.

Now that I think about it again, can we call the field in the struct `dir` instead of `file` as well? Since we're no longer looking for a single file.

Reply:
Thank you for the explanantions. I made the changes,